release-notes/release-notes57.md

```diff
@@ -37,7 +37,17 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
-### Enhancement name
+### The `spellchecker_ignorelist` option now accepts arrays of words per language
+
+{{site.productname}} 5.7 introduces enhancements to the `spellchecker_ignorelist` (formerly `spellchecker_whitelist`, see: [The `spellchecker_whitelist` option has been renamed](#thespellchecker_whitelistoptionhasbeenrenamed)).
+
+- It is now possible to specify arrays of words per language to be ignored by Spell Checker Pro plugin using the `spellchecker_ignorelist` option.
```

Comment:
```suggestion
- It is now possible to specify arrays of words for specific languages to be ignored by Spell Checker Pro plugin using the `spellchecker_ignorelist` option.
```
? this wording feels generally awkward to me. might be best if the sentence is rearranged a bit?

Reply:
It reads okay to me with your changes, @tylerkelly13 any thoughts?